transports/sftp/src/test/java/org/mule/transport/sftp/SftpReceiverRequesterUtilTestCase.java

```diff
@@ -188,6 +193,27 @@ public void testNotReturnStableButNotAgedFile() throws Exception
         assertThat(completedFiles.length, is(0));
     }
 
+    @Test(expected = Throwable.class)
```

Comment:
be more specific. Use the `ExpectedException` rule

Reply:
I'm using a very general exception to ensure that in any failure scenario the sftp client is realised. (In line 203 I am mocking the exception that will be triggered).